### DISCUSSION TITLE
Help my for problem a no zepp os please

### DISCUSSION THREAD

**Discussion by ricardfs** [zepp-health/zeppos-forum#10] [Developer Feedback]
I'm sure this message shouldn't go in this forum but I can't find an answer anywhere and I'd like to see if any of you can help me, my partner has an amazfit bip u pro and I make spheres for him from the zepp creator, obviously in the non-zepp os option, the fact is that I have seen spheres that have icons that can be clicked and take you to the steps, calories or heart rate screen, I imagine it will be with the option of putting an icon or with the option to go to the hot zone but none of these options work for me, please if someone could guide me a bit I would thank you, if the moderators believe that this message should be deleted I apologize for the inconvenience.

**Comment by htoooth** [comment]
@ricardfs 
Hi Ricardfs！You will try to use editable component in then zepp creator.

**Reply by ricardfs** [reply]
Thanks for your answer, I have looked at a clock with Zepp OS and evidently the editable component appears, however in the component bar of clocks that are not Zepp OS this option does not exist.

But I have used dials that do have these editable components for clocks that are not Zepp OS.

Maybe there is some other way to make this possible?

Again I thank you.

**Comment by MyoungXUE** [comment]
@ricardfs The forum welcomes any relevant questions, there is no need to worry about this, thanks for your support of the forum.
